src/utils/rest-api.js

```diff
@@ -939,6 +982,42 @@ export function fetchSensitivityAnalysisResult(studyUuid, currentNodeUuid) {
     return backendFetchJson(url);
 }
 
+export function fetchSensitivityAnalysisProviders() {
+    console.info('fetch sensitivity analysis providers');
+    const url = getSensiUrl() + 'providers';
+    console.debug(url);
+    return backendFetchJson(url);
+}
+
+export function fetchSensitivityAnalysisProvider(studyUuid) {
+    console.info('fetch sensitivity analysis provider');
+    const url = getStudyUrl(studyUuid) + '/sensitivity-analysis/provider';
+    console.debug(url);
+    return backendFetchText(url);
+}
+
+export function updateSensitivityAnalysisProvider(studyUuid, newProvider) {
+    console.info('update sensitivity analysis provider');
+    const url = getStudyUrl(studyUuid) + '/sensitivity-analysis/provider';
+    console.debug(url);
+    return backendFetch(url, {
+        method: 'POST',
+        headers: {
+            Accept: 'application/json',
+            'Content-Type': 'application/json',
+        },
+        body: newProvider,
+    });
+}
+
+export function fetchDefaultSensitivityAnalysisProvider() {
+    console.info('fetch default sensitivity analysis provider');
+    const url =
+        PREFIX_STUDY_QUERIES + '/v1/sensitivity-analysis-default-provider';
```

Comment:
why not calling directly the sensitivity-analysis-server to get the default provider ?